src/hangman/events/impl/letter-guessed.event.ts

```diff
@@ -7,6 +7,6 @@ export class LetterGuessedEvent extends StorableEvent {
 
   constructor(public readonly id: string, public readonly letter: string) {
     super();
-    this.dateModified = new Date();
+    this.dateModified = new Date(); // FIXME: If this is set here, should it not be set in the aggregate root?
```

Comment:
No. Because, this event is created on the user's action. So using that date/time. 
When this event is replayed in the future, it'll still use this dateModified because it's stored in the event. 
If you would set it in the aggregate root, `dateModified` would take the value of the moment the event is replayed